jenkins/catapult/runDockerBuildInnerPrepare.py

```diff
@@ -31,12 +31,6 @@ def main():
 	for name in bin_folder_names:
 		environment_manager.copy_tree_with_symlinks(DATA_VOLUME / 'binaries' / name, USER_HOME / name)
 
-	# LD_LIBRARY_PATH is not passed when llvm-symbolizer is started via asan, so move libs to system location
-	if is_dev_build:
-		system_bin_path = environment_manager.system_bin_path
-		environment_manager.move_glob_with_symlinks(USER_HOME / 'deps', 'libLLVM*', system_bin_path)
-		process_manager.dispatch_subprocess(['ls', '-laF', system_bin_path])
-
```

Comment:
how was this fixed? i.e. why can it be removed?

Reply:
Sanitizer test base image has clang the compiler already installed.  No need to copy compiler files.
https://github.com/symbol/symbol/pull/354/files/3f5d57935b8bb5978627131d4f6b9022404a6f60#diff-579119f97023a7b6e06ea31aaac2573264eac7fd0b2cfe21051fdd638fbbe14a